src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class Claw extends SubsystemBase {
+  private DoubleSolenoid gripSolenoid;
+  private WPI_TalonFX leftClawMotor;
+  private WPI_TalonFX rightClawMotor;
+
+  public Claw() {
+    gripSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, 0, 1);
+    leftClawMotor = new WPI_TalonFX(0);
+    rightClawMotor = new WPI_TalonFX(1);
```

Comment:
The CAN ids and pneumatic ports should be kept in Constants.java. I can show you how to set that up if you need help.